_data/SyaamantakDas/syaamantakdas.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Syaamantak Das
+institution: Indian Institute of Technology Kharagpur
+profile_pic: syaamantakdas(2).jpg 

```

Comment:
The file name is still incorrect, please do one of the following:
1. Change `syaamantakdas(2).jpg` to `syaamantakdas (2).jpg`.
2. Rename `syaamantakdas (2).jpg` to `syaamantakdas.jpg`.
As mentioned before, I'd suggest you rename the file!